testing/trino-product-tests/pom.xml

```diff
@@ -141,32 +156,6 @@
             <artifactId>annotations</artifactId>
         </dependency>
 
-        <dependency>
-            <groupId>org.seleniumhq.selenium</groupId>
```

Comment:
Do we need selenium in root pom.xml? Can we remove it too?

Reply:
Removed